calculate_bmi/solution/calculate_bmi.erl

```diff
@@ -0,0 +1,20 @@
+-module(calculate_bmi).
+
+-export([bmi/1, classify/1]).
+
+-include("../src/person_record.hrl").
```

Comment:
prefer maps to recrods+header files. If you do end up using a record, it's better to encapsulate it in its own module, and always access it through functions. But seriously, maps.

Reply:
[done :)](https://github.com/lambdaclass/erlings/pull/26/commits/61470a3dd16a1e5aa412d28c3d19cff3073d7754)